src/Core/Domain/Product/Customization/CustomizationField.php

```diff
@@ -64,6 +64,7 @@ class CustomizationField
      * @param bool $required
      * @param int|null $customizationFieldId
      * @param bool $addedByModule
+     * @todo: require all fields except the id.
```

Comment:
TODO
- outdated ?
- still valid but need to do it in another PR ?

Reply:
noticed on this, pr but its not in scope, so left todo.
so yes still valid but need to do it in another PR